tests/integration/linux_bridge_test.py

```diff
@@ -86,6 +90,36 @@ def port0_vlan101(port0_up):
         yield state
 
 
+@pytest.fixture
+def bond0(port0_up):
+    bond_name = 'testbond0'
+    port_name = port0_up[Interface.KEY][0][Interface.NAME]
+    bond0 = generate_create_bond_state(bond_name, [port_name])
+    try:
```

Comment:
The `yeld` part is already covered by try/finally in a fixture, you do not need to wrap it.

Reply:
right ! 

After the latest refactor, this no longer applies.